.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "impl/spec-ms"]
+	path = impl/spec-ms
+	url = https://github.com/dhanush-2397/spec-ms.git
+[submodule "impl/ingestion-ms"]
+	path = impl/ingestion-ms
+	url = https://github.com/dhanush-2397/spec-ms.git
```

Comment:
Update this to cQube repo rather than your own fork.

Reply:
Hi 
I have updated it to cQube repo